Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.xaml

```diff
@@ -81,15 +87,70 @@
                                               AutomationProperties.Name="Content"
                                               Content="{TemplateBinding Content}" />
                         </SplitView>
-                        <Button Name="HamburgerButton"
-                                Width="{TemplateBinding HamburgerWidth}"
-                                Height="{TemplateBinding HamburgerHeight}"
-                                Padding="0"
-                                VerticalAlignment="Top"
-                                AutomationProperties.Name="Main button"
-                                Background="Transparent"
-                                BorderThickness="0"
-                                TabIndex="0">
+                        <Button 
+                            Name="HamburgerButton"
+                            Width="{TemplateBinding HamburgerWidth}"
+                            Height="{TemplateBinding HamburgerHeight}"
+                            Padding="0"
+                            VerticalAlignment="Top"
+                            AutomationProperties.Name="Main button"
+                            Background="{TemplateBinding AccentColor}"
```

Comment:
I set the default to be the system accent color in the Style in my latest commit.